### PR TITLE
Fix windows build

### PR DESCRIPTION
**Pull request by tatchi**
I have been able to build the project on my windows machine with these changes.

### CODE REVIEW

**Comment by tatchi** [comment]
Did I break the ci ? 😆

EDIT: From what I see it's only configured to run on master branch

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "@reason-native-web/h1": ">= 1.0.0-alpha.1 < 2.0.0",
     "@reason-native-web/h1-lwt-unix": ">= 1.0.0-alpha.1 < 2.0.0",
     "@reason-native-web/piaf": ">= 1.0.0-alpha.1 < 2.0.0",
-    "@reason-native-web/ssl": ">= 0.5.9-dev.1 < 1.0.0"
+    "@reason-native-web/ssl": "0.5.9002"
```
Comment (yawaramin) [inline package.json]: Can we use a version range here? I'm thinking `">= 0.5.9002 < 1.0.0"`

**Review by yawaramin — COMMENTED** [review]
Wow! Looking forward to merging this one, just one small comment about versioning. Let me know what you think.

File: package.json (hunk 1 above)
Comment (tatchi) [reply package.json]: Yes I guess it should be ok. I just copied what morph uses (fixed version)

Comment (yawaramin) [reply package.json]: Great! Merging.

**Pull request merged**